### DISCUSSION TITLE
Plans to use Ghostty as a terminal emulator in Zed?

### DISCUSSION THREAD

**Discussion by victorhalmagian** [zed-industries/zed#18129] [Feature Requests]
Hei there,

Any plans to use Ghostty terminal emulator instead of Alacritty?

**Comment by SomeoneToIgnore** [comment]
There's nothing on the roadmap albeit it would be nice to be able to use different emulators indeed.
IIRC, it's not that trivial to plug Ghostty into Zed due to the way rendering is happening in both programs, not allowing to unite both worlds — maybe I'm wrong and somebody can come up with the PR.

**Reply by abejfehr** [reply]
`libghostty` was announced [here](https://mitchellh.com/writing/libghostty-is-coming) 2 days after this post was made, that should make it easier

**Reply by tiborvass** [reply]
@SomeoneToIgnore @abejfehr FWIW I have a working branch here https://github.com/tiborvass/zed/tree/libghostty and it's blazingly fast.

**Reply by fosaboi** [reply]
Hello, any news about this ?
